dashboard/package.json

```diff
@@ -20,12 +20,18 @@
   "bugs": {
     "url": "https://github.com/ucberkeley/sis-qa-test-auto/issues"
   },
-  "devDependencies": {
+  "dependencies": {
+    "browserify": "~13.0.0",
```

Comment:
@christianvuerings I have placed the test dependencies into 'devDependencies' since we presumably don't need them in production environments. In production, only the stuff in 'dependencies' need to be installed. What do you think?